webpack.config.js

```diff
@@ -89,17 +91,43 @@ const config = {
           },
         ],
       },
+      // ESM support. See: https://github.com/apollographql/react-apollo/issues/1737#issuecomment-371178602
       {
-        test: /\.json$/,
-        loader: "json-loader",
+        type: "javascript/auto",
+        test: /\.mjs$/,
+        use: [],
       },
     ],
   },
+  optimization: {
+    minimizer: [
+      new UglifyJsPlugin({
```

Comment:
Could have used `mode: production` but this produces much smaller bundles sizes.